package.json

```diff
@@ -2489,6 +2489,316 @@
 											}
 										]
 									},
+									{
+										"key": "languageId:latex",
+										"name": "LaTeX",
+										"type": "bindings",
+										"bindings": [
+											{
+												"key": "=",
+												"name": "+Format",
+												"type": "bindings",
+												"bindings": [
```

Comment:
Just curious and making sure do these built-n format command work for latex doc?

Reply:
Yes they do. The extension has the setup instructions for what is necessary to be setup for it to work correctly. https://github.com/James-Yu/LaTeX-Workshop/wiki/Format